scripts/main.nut

```diff
@@ -42,21 +42,22 @@ function random( min = 0, max = RAND_MAX ) {
 
 dofile("./scripts/scripts.nut")
 
-scripts_load("base/freeroam.nut")
-scripts_load("module/interiors.nut")
-scripts_load("module/vehicle_spawner.nut")
+//scripts_load("base/freeroam.nut")
+scripts_load("base/mallmadness.nut")
+//scripts_load("module/interiors.nut")
+//scripts_load("module/vehicle_spawner.nut")
 scripts_load("module/emote.nut")
-scripts_load("module/teleporter.nut")
+//scripts_load("module/teleporter.nut")
 scripts_load("module/spectate.nut")
```

Comment:
Spectate should be disabled probably. We could have our own spectate mode with different overhead cameras instead.